### PR TITLE
REDHAT-39: Change guide model to add ability handle few paragraphs with text and action.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
What issues does this PR fix or reference?
https://github.com/codenvy/redhat/issues/39

Changed model:
```
{
  "title": "Guide title",
  "sections": [
    {
      "title": "Section #1 :: Simple",
      "paragraphs": [
        {
          "text": "This paragraph contains no action but some text"
        },
        {
          "text": "This paragraph contains no action but some text too"
        }
      ]
    },
    {
      "title": "Section #2 :: Describes how to expand a path in the project explorer",
      "paragraphs": [
        {
          "text": "To open a path the `expand-path` action is used. It requires the mandator `path` parameter containing relative to the project the path to open",
          "action": {
            "label": "Expand path",
            "actionId": "expand-path",
            "parameters": {
              "path": "src/main/java/org/jboss/as/quickstarts/kitchensink/service/MemberRegistration.java"
            }
          }
        }
      ]
    },
    {
      "title": "Section #3 :: Describes how to open a file and run command",
      "paragraphs": [
        {
            "text": "<h4>Open File</h4>"
        },
        {
          "text": "To open a file the `open-file` action is used. It requires the mandatory `file` parameter containing relative to the project the file path to open . Also the auxiliary `line` parameter contains the line number to scroll when file is opened.",
          "action": {
            "label": "Open file",
            "actionId": "open-file",
            "parameters": {
              "file": "src/main/java/org/jboss/as/quickstarts/kitchensink/service/MemberRegistration.java",
              "line": 40
            }
          }
        },
        {
          "text": "<h4>Run command</h4>"  
        },
        {
          "text": "To run a command the `run-command` action is used. It requires the mandatory `name` parameter containing the name of the existed command to run.",
          "action": {
              "label": "Run command",
              "actionId": "run-command",
              "parameters": {
                "name": "KitchenSink2: Build"
              }
          }
          },
          {
          "action": {
              "label": "Run command",
              "actionId": "run-command",
              "parameters": {
                "name": "KitchenSink2: Build"
              }
          }
          }
      ]
    }
  ]
}

```
ScreenShot:
guide

Signed-off-by: Aleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Comment by AndrienkoAleksandr** [comment]
One more example https://gist.github.com/AndrienkoAleksandr/f04cf9a9a17ac90a512b582697cdce82

File: plugins/plugin-quick-start-docs/redhat-plugin-quick-start-docs-shared/src/main/java/com/codenvy/redhat/plugin/quick/start/shared/dto/SubSectionDto.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package com.codenvy.redhat.plugin.quick.start.shared.dto;
+
+import org.eclipse.che.dto.shared.DTO;
+
+@DTO
+public interface SubSectionDto {
```
Comment (tolusha) [inline plugins/plugin-quick-start-docs/redhat-plugin-quick-start-docs-shared/src/main/java/com/codenvy/redhat/plugin/quick/start/shared/dto/SubSectionDto.java]: Name it `Paragraph`

File: plugins/plugin-quick-start-docs/redhat-plugin-quick-start-docs-ide/src/main/java/com/codenvy/redhat/plugin/quick/start/ide/panel/DocsViewPartImpl.java [hunk 2]
```diff
@@ -81,30 +84,42 @@ public DocsViewPartImpl(
...
     DisclosurePanel advancedDisclosure = new DisclosurePanel(sectionDto.getTitle());
     advancedDisclosure.addStyleName(guideResources.getGuideStyle().fullWidthContainer());
-    advancedDisclosure.setContent(createChapter(project, sectionDto));
+    advancedDisclosure.setContent(createSubSectionsWidget(project, sectionDto.getSubSections()));
     return advancedDisclosure;
   }
 
-  private Widget createChapter(Project project, SectionDto sectionDto) {
+  private Widget createSubSectionsWidget(Project project, List<SubSectionDto> subSections) {
+    VerticalPanel horizontalPanel = new VerticalPanel();
```
Comment (tolusha) [inline plugins/plugin-quick-start-docs/redhat-plugin-quick-start-docs-ide/src/main/java/com/codenvy/redhat/plugin/quick/start/ide/panel/DocsViewPartImpl.java]: ^)

**Comment by tolusha** [comment]
@AndrienkoAleksandr 
Attach a new screenshot pls

**Comment by AndrienkoAleksandr** [comment]
@tolusha done.

**Pull request merged**